### ISSUE TITLE
Why collection-repeat is checked?

### ISSUE THREAD

**Issue by guilhermedecampo** [meteoric/meteor-ionic#265]
Hey @nickw,

CollectionRepeat is a solution for huuge lists in angular. Just showing in the screen the items that are visible. Pretty awesome because uses `translate` + 'X | 3D' to improve performance, also the native scroll. 

Unfortunately I don't have a deep knowledge in angular to convert in the blaze way. But I'm willing to help someone who has. 

Maybe this has been solved in another way in Blaze that I don't know..

**Comment by nickw** [comment]
Yeah that shouldn't be checked, I wasn't aware Ionic had that functionality.

**Comment by guilhermedecampo** [comment]
How we can start doing something like in meteor?

https://github.com/driftyco/ionic/blob/master/js/angular/directive/collectionRepeat.js#L2
